cinder/volume.sls

```diff
@@ -113,9 +120,12 @@ hp3parclient:
 
 {%- if backend.engine == 'fujitsu' %}
 
+{%- if grains.get('virtual_subtype', None) not in ['Docker', 'LXC'] %}
```

Comment:
This is valid even for Docker, I would maybe remove test of fujitsu backend as it has special unsatisfiable requirements.